inst/recorder/app.R

```diff
@@ -113,15 +113,40 @@ inputProcessors <- list(
   }
 )
 
+# Add in input processors registered by other packages.
+inputProcessors <- c(inputProcessors, shinytest::getInputProcessors())
+
 # Given an input value taken from the client, return the value that would need
 # to be passed to app$set_input() to set the input to that value.
 processInputValue <- function(value, inputType) {
-  if (is.null(inputProcessors[[inputType]]))
+  if (is.null(inputProcessors[[inputType]])) {
```

Comment:
This seems wrong? Seems like you could do this in an `else` block below (or similar)?

Reply:
The reason it does this check twice is that, the first time it happens, it tries to load the package (like learnr), and that may in turn register an input processor.

The second time it does the check, the target input processor may have been registered.